txaio/aio.py

```diff
@@ -57,8 +57,9 @@
 config = _Config()
 config.loop = asyncio.get_event_loop()
 _stderr, _stdout = sys.stderr, sys.stdout
-_loggers = []  # weak-ref's of each logger we've created before start_logging()
+_loggers = weakref.WeakSet()  # weak-ref's of each logger we've created before start_logging()
```

Comment:
FYI, this broke 2.6 compatibility.
Even if not tested, txaio was supporting 2.6 until 2.2.2.

https://github.com/buildbot/buildbot/pull/2098